dns/src/lib.rs

```diff
@@ -423,6 +427,33 @@ impl Client {
         }
     }
 
+    /// Create a new MDNS client.
+    ///
+    /// # Arguments
+    ///
+    /// * `sn` - The socket number to use for DNS queries.
+    /// * `port` - DNS source port.
+    ///   Use any free port greater than 1024 not in use by other W5500 sockets.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use w5500_dns::{ll::Sn, servers, Client};
+    ///
+    /// const DNS_SRC_PORT: u16 = 45917;
+    ///
+    /// let dns_client: Client = Client::new_mdns(Sn::Sn3);
+    /// ```
+    #[must_use]
+    pub const fn new_mdns(sn: Sn, port: u16) -> Self {
```

Comment:
Does `port` need to be `5353` for clients too?

Reply:
No, and if one tries to use it with regsim it fails, because that port is already bound.